e2e-tests/playwright/support/api/github.ts

```diff
@@ -0,0 +1,48 @@
+import axios, { AxiosResponse } from 'axios';
+import { getOrganizationResponse } from './github-structures';
+import { JANUS_ORG } from '../../utils/constants';
+
+export default class GithubApi {
```

Comment:
@nilgaar There is `githubRequest()` method in APIHelper that also takes care of headers. You may  consider using it.

Reply:
I think that a class that contains both the logic and the urls and that's able to accept parameters on every component of the URL may be more versatile. The advantages that `githubRequest` currently have can be accomplished by a `request.newContext` inside the `GithubApi` class.